Neos.Error.Messages/Classes/Message.php

```diff
@@ -102,10 +111,19 @@ public function getArguments(): array
     }
 
     /**
-     * @return string
+     * @return bool
+     * @api
+     */
+    public function hasTitle(): bool
+    {
+        return $this->title !== null && $this->title !== '';
+    }
+
+    /**
+     * @return string|null
      * @api
      */
-    public function getTitle(): string
+    public function getTitle()
```

Comment:
IMO we have to fix the _calling_ side in this case. Explicitly passing `null` is not in sync with the signature and has been a bug all along (even if one could argue that PHP behaves weird in that case)..
But, anyways, it doesn't really make a difference. Let's just get this fixed and work on greater things :D